src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -60,8 +60,14 @@ protected override void WriteBooleanValue(bool booleanValue)
 
         protected override void WriteDateTimeValue(DateTime dateTimeValue)
         {
-            // ISO 8601            
-            _writer.WriteStringValue(dateTimeValue.ToString("o", CultureInfo.InvariantCulture));
+            // ISO 8601                        
+            _writer.WriteStringValue(dateTimeValue.ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture));

```

Comment:
>"yyyy-MM-ddTHH:mm:ss.fffZ" [](http://example.com/codeflow?start=60&length=26)

nit - since these are used in multiple places, can you define as a const field in the base class?